packages/esm-patient-registration-app/src/patient-registration/field/address/address-hierarchy.component.tsx

```diff
@@ -107,8 +107,8 @@ export const AddressHierarchy: React.FC = () => {
                 <ComboInput
                   key={`combo_input_${index}`}
                   textFieldName={attributes.name}
-                  name={`address.${attributes.name}`}
-                  labelText={t(attributes.name)}
+                  name={`address.${attributes.label}`}
+                  labelText={t(attributes.label)}
```

Comment:
This PR's changes [here](https://github.com/openmrs/openmrs-esm-patient-management/pull/588) by @Piumal1999 were the cause of the issue. 

Let's revert the `name={attributes.name}` change as this will then trigger issues in other distros.

Reply:
Please see the issues when changing the value for the combo box "County". It creates a new object in the Formik state
![Uploading image.png…]()

Reply:
@vasharma05 i have a follow up pr to revert that

Reply:
https://github.com/openmrs/openmrs-esm-patient-management/pull/658